iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergAcidUtil.java

```diff
@@ -243,14 +247,49 @@ public static long getDeleteFilePosition(Record rec) {
     return rec.get(DELETE_FILE_META_COLS.get(MetadataColumns.ROW_POSITION), Long.class);
   }
 
+  private static long hashObjectArray(Object[] values) {
```

Comment:
AFAIK, partition hash is not written anywhere, hence I am assuming this will not have backward compatible issues.